enginetest/queries/column_alias_queries.go

```diff
@@ -168,6 +168,10 @@ var ColumnAliasQueries = []ScriptTest{
 			"insert into uv values (0,3),(3,0),(2,1),(1,2);",
 		},
 		Assertions: []ScriptTestAssertion{
+			{
+				Query:    `select "foo" as dummy, (select dummy)`,
+				Expected: []sql.Row{{"foo", "foo"}},
+			},
```

Comment:
So great to see more of these queries moving out of the "broken" list and working correctly now! Nice work! 😁